community_supplied/Sflow/Sflow-Ingest.playbook

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
There should be separate playbook for each rule as functionality is different. You can club TCP rules in TCP playbook similarly UDP..